### PR TITLE
Bugfix - #902 

### PR DESCRIPTION
**Pull request by v-akarke**

### CODE REVIEW

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

File: module/Entra/AdditionalFunctions/Get-EntraContext.ps1 [hunk 1]
```diff
@@ -1,18 +1,67 @@
 function Get-EntraContext{
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraContext.ps1:1]: Add the MIT license. See this file
https://github.com/microsoftgraph/entra-powershell/blob/main/module/Entra/customizations/Add-EntraGroupOwner.ps1

File: module/EntraBeta/AdditionalFunctions/Get-EntraContext.ps1 [hunk 2]
```diff
@@ -1,15 +1,9 @@
-# ------------------------------------------------------------------------------
-#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
-# ------------------------------------------------------------------------------
-@{
-    SourceName = "Get-AzureADCurrentSessionInfo"
-    TargetName = $null
-    Parameters = $null
-    Outputs = $null
-    CustomScript = @'
+function Get-EntraContext{
```
Comment (KenitoInc) [inline module/EntraBeta/AdditionalFunctions/Get-EntraContext.ps1:1]: Add the MIT license. See this file
https://github.com/microsoftgraph/entra-powershell/blob/main/module/Entra/customizations/Add-EntraGroupOwner.ps1

File: module/Entra/AdditionalFunctions/Get-EntraContext.ps1 [hunk 3]
```diff
@@ -1,18 +1,67 @@
...
+    [CmdletBinding(DefaultParameterSetName = '')]
+    param ()
 
-    PROCESS{
+    PROCESS {
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraContext.ps1:5]: Remove whitespace

**Review by KenitoInc — CHANGES_REQUESTED** [review]

**Review by KenitoInc — APPROVED** [review]

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md [hunk 4]
```diff
@@ -156,9 +158,9 @@ Connect-Entra -UseDeviceCode
 To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code A1B2CDEFGH to authenticate.
 ```
 
-This example shows how to authenticate to Microsoft Entra ID with device.
+This example shows how to interactively authenticate to Microsoft Entra ID using device code flow.
 
-For more information, see [Device Code flow](https://learn.microsoft.com/entra/identity-platform/v2-oauth2-device-code).
+For more information about Device Code flow, see <https://learn.microsoft.com/entra/identity-platform/v2-oauth2-device-code>.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md:163]:
```suggestion
For more information, see [Device Code flow](https://learn.microsoft.com/entra/identity-platform/v2-oauth2-device-code).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md [hunk 5]
```diff
@@ -142,9 +144,9 @@ Connect-Entra -AccessToken $secureString
 Welcome to Microsoft Graph!
 ```
 
-This example shows how to authenticate to Microsoft Entra ID using an access token.
+This example shows how to interactively authenticate to Microsoft Entra ID using an access token.
 
-For more information on how to get or create access token, see [Request an access token](https://learn.microsoft.com/graph/auth-v2-user#3-request-an-access-token).
+For more information on how to get or create access token, see <https://learn.microsoft.com/graph/auth-v2-user#3-request-an-access-token>.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md:149]:
```suggestion
For more information on how to get or create access token, see [Request an access token](https://learn.microsoft.com/graph/auth-v2-user#3-request-an-access-token).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md [hunk 6]
```diff
@@ -176,9 +178,9 @@ Connect-Entra @connectParams
 Welcome to Microsoft Graph!
 ```
 
-This command Connect a session using a ApplicationId and CertificateThumbprint.
+This example shows how to authenticate using an ApplicationId and CertificateThumbprint.
 
-For more information on how to get or create CertificateThumbprint, see [Authenticate with app-only access](https://learn.microsoft.com/powershell/entra-powershell/app-only-access-auth).
+For more information on how to get or create CertificateThumbprint, see <https://learn.microsoft.com/powershell/entra-powershell/app-only-access-auth>.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Connect-Entra.md:183]:
```suggestion
For more information on how to get or create CertificateThumbprint, see [Authenticate with app-only access](https://learn.microsoft.com/powershell/entra-powershell/app-only-access-auth).
```

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**